src/include/stir/recon_buildblock/BinNormalisationFromGEHDF5.h

```diff
@@ -129,6 +129,7 @@ class BinNormalisationFromGEHDF5 :
   float get_dead_time_efficiency ( const DetectionPosition<>& det_pos,
 				  const double start_time, const double end_time) const;
 
+  float get_geometric_factors (int geo_plane_num, Bin uncompressed_bin) const;
```

Comment:
```suggestion
  float get_geometric_factors (int geo_plane_num, const Bin& uncompressed_bin) const;
```

`Bin`s are fairly large, so a reference is best. Note that this signature won't be ok, but we can fix that later once we know what to do!

Reply:
Yup, makes sense to pass as reference, forgot to add it. Fixing. 

 In this case I think its a single Bin, is that actually big?

Reply:
not BIG, but 4 ints (5 for TOF) + 1 float. so 20 bytes. more to come for energy.

who knows what's fasted, but best to use what we use elsewhere of course.

Reply:
@KrisThielemans ah that makes sense, I was expecting around that size. Anyway, still you are right, always pass by reference when one can.